CHANGELOG.txt

```diff
@@ -21,6 +21,7 @@ Changelog
  * Add ability to select multiple items at once within bulk actions selections when holding shift on subsequent clicks (Hitansh Shah)
  * The upgrade notification, shown to admins on the dashboard if Wagtail is out of date, will now show a more suitable version to update to, not just the latest patch (Tibor Leupold)
  * Upgrade notification can now be configured to only show updates when there is a new LTS available via `WAGTAIL_ENABLE_UPDATE_CHECK = 'lts'` (Tibor Leupold)
+ * Images will now only load once they are visible on screen (Jake Howard)
```

Comment:
I don't think we should, no. This can make a huge performance improvement for site users, and applying it in blanket is very simple to do (in fact, scoping it to the admin is more complex), and should have no negative impact for users, just positive ones. It could make hero images take marginally longer to load, but that should be offset by the reduced load from other images on the page to the point of not making any real difference.

Reply:
This seems like a welcome change to me but I think it would only be acceptable as a blanket change if it’s admin-only, or opt-in only. For hero images, this will make them take longer to load since they might get de-prioritised until the whole HTML is done receiving (or even parsed / DOM loaded? not sure). It’ll significantly increase LCP in particular.

if we want to scope this to the admin only, worst case we could add loading=lazy to all our image tags ?